frontend/src/types.ts

```diff
@@ -54,6 +54,11 @@ export interface User {
   expiresAt: string;
 }
 
+export interface ErrorResponse {
+  error: boolean;
```

Comment:
Why do we return an error boolean? what is the purpose? the endpoint already returns a status code and you can use `response.ok` to check that